core-java-modules/core-java-string-operations-2/src/test/java/com/baeldung/contains/CaseInsensitiveWorkaroundsUnitTest.java

```diff
@@ -0,0 +1,53 @@
+package com.baeldung.contains;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.regex.Pattern;
+
+/**
+ * BAEL-3739: Different ways to solve the contains() case insensitive behavior.
+ */
+public class CaseInsensitiveWorkaroundsUnitTest {
+
+    private String src = "Lorem ipsum dolor sit amet";
+    private String dest = "lorem";
+
+    @Test
+    public void givenString_whenCallingContainsWithToLowerOrUpperCase_shouldReturnTrue() {
```

Comment:
```suggestion
    public void givenString_whenCallingContainsWithToLowerOrUpperCase_thenReturnsTrue() {
```

Reply:
Totally agree with, I accepted the proposed changes